diff --git a/dist/shaka-player.compiled.d.ts b/dist/shaka-player.compiled.d.ts
index 3ba6bf60570343ed3c0912ff33bdb2294522b0fa..e932d4c4ad1ba01b9b06ebb547146e2b931829ef 100644
--- a/dist/shaka-player.compiled.d.ts
+++ b/dist/shaka-player.compiled.d.ts
@@ -5613,3 +5613,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.ui.d.ts b/dist/shaka-player.ui.d.ts
index f6b803e61c481c6dc74c5d3944e90557f79c1250..4fbfd10ee20dda4beca093cb18749eda2761ba81 100644
--- a/dist/shaka-player.ui.d.ts
+++ b/dist/shaka-player.ui.d.ts
@@ -6368,3 +6368,5 @@ declare namespace shaka.extern {
 declare namespace shaka.ui.Watermark {
   type Options = { alpha : number , color : string , displayDuration : number , interval : number , position : string , size : number , skip : number , text : string , transitionDuration : number , type : string } ;
 }
+
+export default shaka;
