diff --git a/dist/shaka-player.compiled.d.ts b/dist/shaka-player.compiled.d.ts
index 55092e9ba315631d00850773eb84a9fa7fc2b09a..601eb2642c9a288a5ab9901d4afab60112cae1b9 100644
--- a/dist/shaka-player.compiled.d.ts
+++ b/dist/shaka-player.compiled.d.ts
@@ -5338,3 +5338,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.ui.d.ts b/dist/shaka-player.ui.d.ts
index b655be09cad62c2099a34ea8a67b4e8a7ffccad4..b9ca3424f30b5d7eb173e5df34f1fb98f8b10cde 100644
--- a/dist/shaka-player.ui.d.ts
+++ b/dist/shaka-player.ui.d.ts
@@ -6084,3 +6084,5 @@ declare namespace shaka.extern {
 declare namespace shaka.ui.Watermark {
   type Options = { alpha : number , color : string , displayDuration : number , interval : number , position : string , size : number , skip : number , text : string , transitionDuration : number , type : string } ;
 }
+
+export default shaka;
diff --git a/index.d.ts b/index.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..86130b219f20157404545f817f41485ab13ced25
--- /dev/null
+++ b/index.d.ts
@@ -0,0 +1,2 @@
+/// <reference path="./dist/shaka-player.compiled.d.ts" />
+/// <reference path="./dist/shaka-player.ui.d.ts" />
diff --git a/ui.d.ts b/ui.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..2448bf7128d5d9a66fbcf44da8b5e4e9b423a13d
--- /dev/null
+++ b/ui.d.ts
@@ -0,0 +1,3 @@
+import shaka from 'shaka-player/dist/shaka-player.ui';
+export * from 'shaka-player/dist/shaka-player.ui';
+export default shaka;
